src/features/controls.ts

```diff
@@ -353,6 +353,9 @@ export const ControlsMixin = (ModelViewerElement:
         [$onResize](event: any) {
           super[$onResize](event);
           this[$updateCamera]();
+          // Render a frame right away
+          this[$scene].isDirty = true;
+          this[$scene].renderer.render(performance.now());
```

Comment:
If I didn't apply the `ControlsMixin` to the base element, what would happen when we resize?

Reply:
Good point, I wasn't thinking about using them independently.

Reply:
Is it possible that we don't even need the call to `renderer.render` here? It seems like the parent implementation of `[$onResize]` will cause us to re-render eventually..

Reply:
The eventually part was exactly the problem; @mrdoob had a better explanation in #555.

Reply:
Yes, I see. It seems like we should be invoking this in the base class (not in the mixin) because the problem occurs regardless of whether `ControlsMixin` is applied. Is that correct?

Reply:
That's how it was in #555, but the problem there is that it gets redrawn without being reframed, since that happens in ControlsMixin.

Reply:
It seems like we might be trading one timing problem for another (unintentionally). `[$needsRender]()` causes `isDirty` to be set. Then, you call `render` directly in the mixin implementation causing an immediate render (and `isDirty` gets reset). So, on the next frame, the scene may not be re-rendered as intended.

Perhaps this timing issue could be solved by invoking `[$udpateCamera]()` before calling the super implementation of this method? Then we only need to render synchronously in one place (the base implementation).

Reply:
Hmm, I just tried this, only to realize that [$updateCamera] is dependent on the base class setting the width and height first, so now we're trying to call the base class both before and after. Should I just setSize twice?